src/test/resources/application.properties

```diff
@@ -6,13 +6,11 @@ management.endpoints.web.path-mapping.health=/healthcheck
 uk.gov.companieshouse.documentsigningapi.signpdf=/document-signing/sign-pdf
 
 # Environment variables
-environment.keystore.type=${KEYSTORE_TYPE}
-environment.keystore.path=${KEYSTORE_PATH}
-environment.keystore.password=${KEYSTORE_PASSWORD}
-environment.certificate.alias=${CERTIFICATE_ALIAS}
-
-environment.signed.doc.bucket.name=${SIGNED_DOC_BUCKET_NAME}
-
-environment.coversheet.images.path=${COVERSHEET_IMAGES_PATH:}
+environment.keystore.type=pkcs12
+environment.keystore.path=src/test/resources/keystore.p12
+environment.keystore.password=password
+environment.certificate.alias=dockerkeystore
+environment.signed.doc.bucket.name=document-signing-api
+environment.coversheet.images.path=src/main/resources/coversheet
 
 spring.main.allow-bean-definition-overriding=true
```

Comment:
Try to get rid of this if possible, appreciate it's not a new addition but it's not ideal to have enabled. If it's handling conflicting beans from 3rd parties you can keep it enabled

Reply:
Using variables for some reason doesn't reach the integration test. Removing this will cause beans to not be created during itest.

Reply:
Sorry, talking exclusively about `spring.main.allow-bean-definition-overriding=true`

Reply:
Turning it off is creating spring bean creation errors and having it on resolves this issue.